### DISCUSSION TITLE
How to determine if scoop is installed?

### DISCUSSION THREAD

**Discussion by eggbean** [ScoopInstaller/Scoop#5760] [Help and Questions] [answered]
What's the best way to determine if scoop is installed or not? I have a few script which fail when scoop isn't install yet, as they use various programs provided by scoop, so I want the scripts to exit when scoop is not installed.

I would like to know this for cmd.exe and powershell. Thanks.

**Comment by hrxn** [comment, marked as answer]
CMD
```
where scoop
```

PS
```
gcm scoop
```

**Comment by eggbean** [comment]
I'm currently trying to figure out how to install scoop for all users. There isn't any clear information on doing this, but what I have found involves copying the scoop directory and then `scoop uninstall scoop`, which has removed all this shims and now I cannot use the scoop provided programs. This is what led me to ask this, but the commands you suggest are result as true even though scoop is "uninstalled".

**Comment by eggbean** [comment]
I see that the scoop directory has been recreated in `C:\ProgramData\scoop` which is what I set `%SCOOP%` to be, so I guess it's the only answer possible. Cheers.
